app/classifier/tasks/dropdown/index.cjsx

```diff
@@ -121,24 +120,30 @@ module.exports = React.createClass
   handleFocus: ->
     @menus[0].focus()
 
-  getOptions: (i, optionsKeys) ->
-    optionsKey = if optionsKeys.length then optionsKeys.join(';') else '*'
+  getOptionsKey: (i, optionsKey = '') ->
+    {selects} = @props.task
+    select = selects[i]
+    [condition] = selects.filter (filterSelect) => filterSelect.id is select.condition
+    conditionIndex = selects.indexOf(condition)
+    optionsKey = if optionsKey.length is 0 then @props.annotation.value[conditionIndex]?.value else "#{@props.annotation.value[conditionIndex]?.value};#{optionsKey}"
```

Comment:
There are two tests that fail when state is used:
1. `DropdownTask with multiple selects and all annotations provided, no custom answers, should have the supplied annotations selected`
2. `DropdownTask with multiple selects and all annotations provided, including custom answers, should have the supplied annotations selected`
I think due to if `annovation.values` has answers already (clicking back, or persist annotations option enabled), `syncAnnotations` runs before mount, or the state has a chance to update.